docs/docs/behaviors/macros.md

```diff
@@ -62,6 +78,32 @@ bindings
 There are a set of special macro controls that can be included in the `bindings` list to modify the
 way the macro is processed.
 
+### Parameters
+
+When creating a macro that takes parameter(s), there are control when the parameters passed to the macro are used
+within the macro itself. All of the controls are "one shot" and will change how the passed in parameters are used for the very next behavior in the `bindings` list of the macro.
```

Comment:
Might be useful to clarify that it applies to the next non-macro control behavior, so `&macro_param1to1 &macro_param2to2 @macro_press &foo 0 0` is a valid thing that works.

Reply:
Tweaked the wording here.